db/version_set.cc

```diff
@@ -1875,14 +1876,16 @@ void Version::Get(const ReadOptions& read_options, const LookupKey& k,
   // need to provide it here.
   bool is_blob_index = false;
   bool* const is_blob_to_use = is_blob ? is_blob : &is_blob_index;
+  std::unique_ptr<BlobFileMergeCallback> blob_file_merge_callback(
```

Comment:
You could simply allocate this on the stack. No need to use `unique_ptr`; you can simply pass it to `GetContext` via a raw (non-owning) pointer.